### PR TITLE
Feature: Remove admission state (EXPOSUREAPP-13914, EXPOSUREAPP-13916)

### PR DESCRIPTION
**Pull request by mlenkeit**
This PR changes CCL to not return any admission state anymore.

### CODE REVIEW

**Comment by Ein-Tim** [comment]
@mlenkeit If the team is planning to remove the admission state (3G, 2G, etc.), I strongly suggest to clearly communicate this to the user. If we don't want the user to think that they have lost their admission state for some reason, in-app information should be added on why the admission state is not displayed anymore.

**Comment by Ein-Tim** [comment]
Thanks Maximilian!

File: lib/ccl/ccl-constants.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-export const TC_PCR_VALIDITY_IN_HOURS = 72
-export const TC_RAT_VALIDITY_IN_HOURS = 48
+export const TC_PCR_VALIDITY_IN_HOURS = 24
+export const TC_RAT_VALIDITY_IN_HOURS = 24
```
Comment (Ein-Tim) [inline lib/ccl/ccl-constants.js:2]: Shouldn't this also be rolled back to the app's receive test results feature?

File: resources/i18n/de/strings.xml [hunk 2]
```diff
@@ -81,9 +81,9 @@
   <!-- XTXT: Admission State - OTHER Badge Text -->
   <string name="ADMISSION_STATE_OTHER_BADGE_TEXT">""</string>
   <!-- XTXT: Admission State - OTHER Card Body -->
-  <string name="ADMISSION_STATE_OTHER_LONG_TEXT">"Ihre Zertifikate erfüllen derzeit keine Regel."</string>
+  <string name="ADMISSION_STATE_OTHER_LONG_TEXT">"Der G-Status ist zur Zeit nicht relevant und wird daher nicht von der App ausgewiesen."</string>
```
Comment (KathrinLuecke) [inline resources/i18n/de/strings.xml:84]: How about: "Zertifikate werden zur Zeit nicht geprüft und Ihr Status daher nicht von der App ausgewiesen."

Comment (MikeMcC399) [reply resources/i18n/de/strings.xml:84]: @KathrinLuecke 

> "Zertifikate werden zur Zeit nicht geprüft und Ihr Status daher nicht von der App ausgewiesen."

Wouldn't that statement be misleading if certificates are being checked to find out whether mask wearing is mandatory?

**Pull request merged**